### ISSUE TITLE
Assaultcom, Starcom, Supreme Com Buildmenu suggestions

### ISSUE THREAD

**Issue by Zonkspring** [n3wm1nd/TA#904]
Those guys are pretty cool, but their Build menus can at times seem... ah, shall we say rather conservative? I think it couldn't hurt if those guys could build a bit more advanched stuff. 
By the time u have Assaultcom u 95% have T2 already anyways.

So here it goes:

LLT-replacements:
- Battlecom can already build rapid-llt without helper, so:
 Assault com Beamer, Starcom Pitbull, Supreme com Anni (regular one)

Wind, Tidals, Mex, Metalmakers, Fusions, Solars, Storages, Research Centers, Cons:
- Assault com and up get Adv. Wind, Adv. Tidals, Adv. Solars. Not their job to build those anyways but might at least give them the Adv. versions anyways, just seems logical.
- Starcom and Supreme Com get Moho Mex (regular and UW) and Moho-Metalmakers (regular and UW). (However, giving SupCom Dildo would make that morph OP.)
- Assault com gets Light Fusion instead of Driller, Starcom regular Fusion, Supremecom Advfus.
- Assault and up build T2 Storages instead of T1
- Starcom Builds T2 Research center, Supreme Com T3.
- Starcom and Supreme com Build T3 Vehcon instead of EcoCons.

Radar, Sonar, Torpedo, AA:
- Battle 1.5 Radar, Assault and up T2 Radar
- Star and Sup T2 Sonar, Adv. Torpedo, NS Flak
- Battle already builds upg Defender, so:
Assault Packo, Starcom and Supcom Flak

Along with that I'd give some Buildpower increases to Starcom (600 -> 1000) and SupremeCom (650 -> 2000). Assaultcom no increase, this guy gets enough stuff with his morph already.

Not a change thats super-urgent imo, just something that would fit the general theme of Commanders remaining useful and relevant into lategame in Tech Annihiliation.
It always seems a bit silly when a Supcom can only make T1.5 Mexes.

**Comment by skymyj** [comment]
I'm agree with this.
Just a lot of work :) to change all menu for arm/core/tll.

Nix, if you are agree, I will draw a Menu. we could vote to add/delete some units/structures.

**Comment by Zonkspring** [comment]
Yes, a lot of work. :) Not the most important change that needs to be done fast-fast I think, but would be cool. :+1:

**Comment by SoulRaiZor** [comment]
Sounds good, yes.
But no anni or pitbull to assault com plz

**Comment by skymyj** [comment]
Assault Com =
**Adv Solar
**Adv Tidal
**Adv WindMill
Driller
Adv Metal Maker
Sub Metal Maker
M/E Storage
M/E Sub Storage
Adv Metal Extractor (delete classic sub metal extractor)
Kbot Lab
Veh Lab
Air Lab
Naval Lab
Adv Radar (t1.5)
Sonar
Teeth
Sub Teeth
** Adv HLT
Sub HLT
Torpedo
armdl
** SAM (AA)
Naval AA
Jump Pad
Helper
Builder
Armpcan (pewwee/AK canon)
=> what do you think ?

**Comment by skymyj** [comment]
Star Com = 
** Medium Fusion (+500E)
** Adv Metal Maker
** Metal Extractor T2
** Sub Metal Extractor T2
** M/E T2 Storage
** Sub M/E T2 Storage
** T2 Radar
** T2 Sonar
** T2 lab kbot
** T2 lab Veh
** T2 lab Air
** Hardened Nano
** wall
** Pop-up Gauss 
** Flak
** Sub Flak
Jump Pad
Helper
Builder
Armpcan (pewwee/AK canon)

**Comment by nixtux** [comment]
What do the * mean, want you want to add?

**Comment by skymyj** [comment]
Sorry. I would say : changes with actually
*=The unit changed

**Comment by skymyj** [comment]
Supreme Commander
** Fusion (+1000E)
** Adv Metal Maker
** Metal Extractor T2
** Sub Metal Extractor T2
** M/E T2 Storage
** Sub M/E T2 Storage
** T2 Radar
** T2 Sonar
** T2 lab kbot
** T2 lab Veh
** T2 lab Air
T4 lab Kbot
** Hardened Nano
** wall
** Pop-up Gauss 
** Tachion
** Adv Torpedo
** Flak
** Sub Flak
Jump Pad
Helper
Builder
Armpcan (pewwee/AK canon)
Tech Center

**Comment by Zonkspring** [comment]
+1 To all of those Skymyj :)

**Comment by skymyj** [comment]
I begin with Arm faction.

**Comment by skymyj** [comment]
I will test in local & I will show a screenshot for validation.

**Comment by skymyj** [comment]
After testing : 
Assaut Corcom
![corcom assault](https://cloud.githubusercontent.com/assets/2990630/12487992/c9a8e32a-c069-11e5-8e57-44264a458dd5.png)
Star Corcom![corcom star](https://cloud.githubusercontent.com/assets/2990630/12487994/c9aa258c-c069-11e5-9f2b-a86e92420d82.png)
Supreme Corcom

![armcom supreme](https://cloud.githubusercontent.com/assets/2990630/12487993/c9a90ee0-c069-11e5-9747-9cee4cbdaea1.png)

**Comment by skymyj** [comment]
Assault Armcom
![assault armcom](https://cloud.githubusercontent.com/assets/2990630/12488589/a2997fb2-c06c-11e5-9ac7-7427c2a1dbf1.png)
Star Armcom
![armcom star](https://cloud.githubusercontent.com/assets/2990630/12488591/a29f3074-c06c-11e5-83d4-c89221f9cabd.png)
Supreme Armcom
![supreme armcom](https://cloud.githubusercontent.com/assets/2990630/12488590/a29b2240-c06c-11e5-964a-459f3fa8c600.png)

**Comment by skymyj** [comment]
TLL Assault Com
![assault armcom](https://cloud.githubusercontent.com/assets/2990630/12510812/e0dc7360-c10c-11e5-8c6d-916db041429e.png)
TLL Star Com
![armcom star](https://cloud.githubusercontent.com/assets/2990630/12510814/e100a780-c10c-11e5-8cc2-18d2644a1ee1.png)
TLL Supreme Com
![supreme armcom](https://cloud.githubusercontent.com/assets/2990630/12510813/e0ff1f32-c10c-11e5-90c2-6264617ee453.png)

**Comment by skymyj** [comment]
All tested. 
Say me if you are agree with this.

**Comment by nixtux** [comment]
Skymyj can you give me a nice readable list of what you added /removed /replaced please

**Comment by skymyj** [comment]
Text or visual ?

**Comment by skymyj** [comment]
Added for 3 factions : 

For Assault Com : 

Added : Adv Solar, Adv Wind, Adv Tidal, Adv HLT, SAM, T2 Tech Center
Deleted : Solar, Tidal, Wind, Tidal, T1 Tech Center

Exception TLL : T2 cons remplace T1 Cons. TllBurner remplaces Rockos

**Comment by skymyj** [comment]
Close Subject maybe. 
Commits Merged
